app/views/organizations/staff/default_pet_tasks/index.html.erb

```diff
@@ -30,6 +31,9 @@
                   <td>
                     <%= task.description %>
                   </td>
+                  <td>
+                    <%= task.species %>
```

Comment:
```suggestion
                    <%= task.human_enum_name(:species) %>
```
We have a method to translate enums. Update `en.yml` :
```
activerecord:
  attributes:
    default_pet_tasks:
      species:
        All: "All"
        .......
```

Reply:
OK

Reply:
I18n::InvalidPluralizationData at /staff/default_pet_tasks/13/edit
translation data {:All=>"All", :Dog=>"Dog", :Cat=>"Cat"} can not be used with :count => 1. key 'one' is missing. Due to the above change I'm getting this error in the bootstrap_form in default_pet_tasks form partial :species select.  
`  Model.human_attribute_name(:species) `  on enums returns same error which i think bootstrap_form_with internally works using this.

Reply:
> I18n::InvalidPluralizationData at /staff/default_pet_tasks/13/edit translation data {:All=>"All", :Dog=>"Dog", :Cat=>"Cat"} can not be used with :count => 1. key 'one' is missing. Due to the above change I'm getting this error in the bootstrap_form in default_pet_tasks form partial :species select. ` Model.human_attribute_name(:species)` on enums returns same error which i think bootstrap_form_with internally works using this.

.

Reply:
`<%= f.select :species,
                     options_for_select(DefaultPetTask.species.keys, task.species),
                     { prompt: t('general.please_select'), label: "Species" },
                     { class: 'form-control' } %>` . We can have some fix like this by passing label option which i feel is not satisfactory. But still Model.human_attribute_name is a normally used rails method which is failing for enum attributes which have been added as part of human_enum_name in YAML files. We can check for existing keys in console. This is my dilemma. Would like to hear your thoughts.

Reply:
In `en.yml` under species add `one: "Species"`

I think we should also add a new method that returns all the translations for a specific model enum. It would simplify the options for select.
```
<%= f.select :species,
        options_for_select(DefaultPetTask.human_enum_names(:species), task.human_enum_name(:species)), 
       { prompt: t('general.please_select') }, { class: 'form-control' } %>
```

In `application_record.rb` we could do something like
```
  def self.human_enum_names(enum)
    enum_i18n_key = enum.to_s
    names = []
    send(enum).each_key do |key|
      names << I18n.t("activerecord.attributes.#{model_name.i18n_key}.#{enum_i18n_key}.#{key}")
    end
    names
  end
```
Feel free to adjust it.